src/components/calcite-button/calcite-button.tsx

```diff
@@ -108,6 +108,22 @@ export class CalciteButton {
       .reduce((acc, { name, value }) => ({ ...acc, [name]: value }), {});
   }
 
+  // if a button has an associated form, create a proxy button and click it
+  private handleClick = (e: Event) => {
+    const requestedForm = this.el.getAttribute("form");
+    const targetForm = requestedForm
+      ? document.querySelector(`#${requestedForm}`)
+      : this.el.closest("form");
+    if (targetForm) {
+      const proxyElement = document.createElement("button");
```

Comment:
The proxy seems kind of feeble :|

Reply:
I took it right from Stencil themselves 😄 - just added the check for associated forms for the use case of buttons outside of containing forms. Open to suggestions there for improvement.